### PR TITLE
Vegemite 2: Australian Boogaloo

### PR DESCRIPTION
**Pull request by Pacmandevil**
Vegemite.

### CODE REVIEW

**Comment by Arrow768** [comment]
Changes that do not fix a ongoing issue in master generally go into development

**Comment by KepQwet** [comment]
```
 __     __  ________   ______   ________  __       __  ______  ________  ________ 
|  \   |  \|        \ /      \ |        \|  \     /  \|      \|        \|        \
| $$   | $$| $$$$$$$$|  $$$$$$\| $$$$$$$$| $$\   /  $$ \$$$$$$ \$$$$$$$$| $$$$$$$$
| $$   | $$| $$__    | $$ __\$$| $$__    | $$$\ /  $$$  | $$     | $$   | $$__    
 \$$\ /  $$| $$  \   | $$|    \| $$  \   | $$$$\  $$$$  | $$     | $$   | $$  \   
  \$$\  $$ | $$$$$   | $$ \$$$$| $$$$$   | $$\$$ $$ $$  | $$     | $$   | $$$$$   
   \$$ $$  | $$_____ | $$__| $$| $$_____ | $$ \$$$| $$ _| $$_    | $$   | $$_____ 
    \$$$   | $$     \ \$$    $$| $$     \| $$  \$ | $$|   $$ \   | $$   | $$     \
     \$     \$$$$$$$$  \$$$$$$  \$$$$$$$$ \$$      \$$ \$$$$$$    \$$    \$$$$$$$$
                                                                                  
                                                                                  
                                                                              
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -157,6 +157,20 @@
 	..()
 	M.intoxication += removed*0.02 //Very slightly alcoholic
 
+/datum/reagent/nutriment/coating/vegemite
+	name = "Vegemite"
+	cooked_name = "burnt Vegemite"
+	id = "vegemite"
+	color = "#0c0901"
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: Are you sure that colour is correct ? - Its almost black

Comment (Pacmandevil) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: I googled it - and I think it looks accurate.

Comment (CommanderXor) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: Vegemite is black in the jar at high concentrations. When spread out on bread, and so fourth, it's brown.

**Comment by skull132** [comment]
\> Not implemented in any way
yeeee give me a reason not to close this.

**Comment by Pacmandevil** [comment]
see skull, you're implying that this wasn't a meme PR

**Pull request closed without merging**